.vitepress/config.ts

```diff
@@ -626,13 +626,6 @@ export default defineConfigWithTheme<ThemeConfig>({
         defer: ''
       }
     ],
-    [
```

Comment:
no need to remove this script here - the banner injection is controlled externally and will be off during that period